src/main/webapp/pages/samples/sample_files.html

```diff
@@ -182,6 +182,58 @@ <h4 class="media-heading" th:text="#{samples.files.empty.title}"></h4>
 			</table>
 		</div>
 	</div>
+
+	<div class="row" th:if="${not #lists.isEmpty(fast5)}">
+		<div class="col-md-12">
+			<h2 th:text="#{sample.files.fast5}">FAST5 Files</h2>
+			<table class="table fast5" th:with="dateFormat=#{locale.date.short}, url=@{${#httpServletRequest.requestURI}}">
+				<tr class="fast5_row" th:each="file : ${fast5}">
+					<td class="td_icon fast5_icon">
+						<i class="far fa-file-alt"></i>
+					</td>
+					<td>
+						<div class="file-name td_info--value"
+							 style="display: flex; align-items: center; height: 35px;">
+							<div th:text="${file.label}"></div>
```

Comment:
Need the link to the fastqc results here.  It's exaclty the same as for sequence files.

Reply:
Updated in 82d3f5f5397234fcc0b347bf1e8f669743627243